pkg/volume/flexvolume/probe_test.go

```diff
@@ -342,16 +342,17 @@ func TestProberMultiThreaded(t *testing.T) {
 
 	// Act
 	for i := 0; i < 100; i++ {
+		wg.Add(1)
 		go func() {
 			defer wg.Done()
+			pluginNameMutex.Lock()
```

Comment:
I don't have context to know if this change is still testing what we really want to test ... is locking the mutex outside the Probe call still effectively testing what we intend?